src/content/docs/en/concepts/why-astro.mdx

```diff
@@ -1,45 +1,63 @@
 ---
 title: Why Astro?
-description: "Astro is an all-in-one web framework for building fast, content-focused websites. Learn more."
+description: "Astro is an all-in-one web framework for building fast, content-driven websites. Learn more."
 i18nReady: true
 ---
 
-Astro is an **all-in-one** **web framework** for building **fast,** **content-focused websites.**
+**Astro** is a web framework for building **content-driven websites** that load fast with great SEO.
 
-Why choose Astro over another web framework? Here are five core design principles to help explain why we built Astro, the problems that it exists to solve, and why Astro may be the best choice for your project or team.
+:::tip
+Explore the [collection of Astro themes](https://astro.build/themes/) for blogs, portfolios, docs, landing pages, SaaS, marketing, ecommerce sites, and more!
+::: 
+
+Astro is best known for pioneering and popularizing [**islands**](/en/concepts/islands/): a JavaScript frontend architecture optimized for performance, flexibility, and simplicity.
+
+## Features
+
+Astro includes everything you need built-in to get started quickly, with hundreds of available integrations for added features, performance, and developer experience. 
+
+Some highlights include:
 
-#### Astro is...
+- **[Islands:](/en/concepts/islands/)** A component-based web architecture optimized for content-driven websites.
+- **[UI-agnostic](/en/core-concepts/framework-components/):** Supports React, Preact, Svelte, Vue, Solid, Lit, HTMX, web components, and more.
+- **[Server-first:](/en/core-concepts/rendering-modes/)** Moves expensive rendering off of your visitors' devices.
+- **[Zero JS, by default:](/en/core-concepts/astro-components/)** Less client-side JavaScript to slow your site down.
+- **[Content collections:](/en/guides/content-collections/)** Organize, validate and provide TypeScript type-safety for your Markdown content.
+- **[Customizable:](/en/guides/integrations-guide/)** Tailwind, MDX, and hundreds of integrations to choose from.
```

Comment:
The formatting is a bit inconsistent here, sometimes the `:` is part of the link, sometimes not. I would change so that the `:` isn't part of the link, but is still bold.

```suggestion
- **[Islands](/en/concepts/islands/):** A component-based web architecture optimized for content-driven websites.
- **[UI-agnostic](/en/core-concepts/framework-components/):** Supports React, Preact, Svelte, Vue, Solid, Lit, HTMX, web components, and more.
- **[Server-first](/en/core-concepts/rendering-modes/):** Moves expensive rendering off of your visitors' devices.
- **[Zero JS, by default](/en/core-concepts/astro-components/):** Less client-side JavaScript to slow your site down.
- **[Content collections](/en/guides/content-collections/):** Organize, validate, and provide TypeScript type-safety for your Markdown content.
- **[Customizable](/en/guides/integrations-guide/):** Tailwind, MDX, and hundreds of integrations to choose from.
```

Reply:
Yes, exactly what I would have chosen. :one: :brain: